Makefile

```diff
@@ -58,4 +58,4 @@ gemuse:
 update-alerts:
 	cd tools/springboard-alerts && \
 	bundle install && \
-	bundle exec ruby update-alerts.rb
+	bundle exec ruby update-alerts.rb "$(ROOT_DIR)"
```

Comment:
Let's change this to `XCODE_CORE_SIM_DIR`

And add a code comment to the make rule with an example of an XCODE_CORE_SIM_DIR.